docs/content/integration/proxies/forwarded-headers/index.md

```diff
@@ -70,11 +70,18 @@ achieve a secure result. Please see the [Cloudflare] documentation on
    below).
 9. Set the `Then` section dropdown to `Remove`.
 10. Set the `Then` section `Header name` to `X-Forwarded-For`.
-11. Click `Save`.
+11. Click `Deploy`.
 
 {{< figure src="cloudflare_1.png" alt="Steps 1 to 4" width="736" >}}
+
+Steps 5-11: IP Source Address | is not in | *list of trusted source addresses*
+
 {{< figure src="cloudflare_2.png" alt="Steps 5 to 11" width="736" >}}
 
+Steps 5-11: X-Forwarded-Fo | does not equal | *blank*
+
+{{< figure src="cloudflare_3.png" alt="Steps 5 to 11" width="736" >}}
+
```

Comment:
It seems the third image is a duplicate of the second image (cloudflare_2.png) 
```suggestion
```

Reply:
There are two methods, either "Remove When Not From Trusted Source" and "Always Remove." The table at the bottom shows the options, but there was initially no image for the, "Generally speaking the `Always Remove` option is the correct option." method. At first I had image two just be the Always Remove option, but James suggested I instead add a third image.

Reply:
I missed that subtle difference.

In that case lets make it clear that there are two different options in the caption/title/alt text.